extensions/notebook/src/common/notebookUtils.ts

```diff
@@ -19,17 +19,9 @@ export class NotebookUtils {
 
 	constructor() { }
 
-	public async newNotebook(connectionProfile?: azdata.IConnectionProfile): Promise<azdata.nb.NotebookEditor> {
+	public async newNotebook(options?: azdata.nb.NotebookShowOptions): Promise<azdata.nb.NotebookEditor> {
 		const title = this.findNextUntitledEditorName();
 		const untitledUri = vscode.Uri.parse(`untitled:${title}`);
-		const options: azdata.nb.NotebookShowOptions = connectionProfile ? {
```

Comment:
Why are you removing this? That seems unrelated to your changes

Reply:
I changed the parameter type from azdata.IConnectionProfile to azdata.nb.NotebookShowOptions so that I can pass in initialContent. This is for creating the notebook that helps users reinstall pip packages.